### PR TITLE
Bugfix stack overflow

### PR DESCRIPTION
**Pull request by sgaragagghu**

### CODE REVIEW

File: client/storage.c [hunk 1]
```diff
@@ -491,7 +491,7 @@ void xdag_init_storage(xdag_time_t start_time, xdag_time_t end_time, void *data,
 /* Calls a callback for all blocks from the repository that are in specified time interval; returns the number of blocks */
 uint64_t xdag_load_blocks(xdag_time_t start_time, xdag_time_t end_time, void *data, void *(*callback)(void *, void *))
 {
-	struct xdag_block buf[bufsize], *pbuf[bufsize];
+	struct xdag_block *buf, *pbuf[bufsize];
```
Comment (xrdavies) [inline client/storage.c]: Did you meet no memory issue with this section?

Comment (sgaragagghu) [reply client/storage.c]: Stack size depend system, on some system xdag crash for this.

Comment (xrdavies) [reply client/storage.c]: Could you point out which system? I would like know the specific details before confirm code changes.

Comment (sgaragagghu) [reply client/storage.c]: OpenBSD 6.3

**Pull request merged**